src/defaultTranslations/relative.time.ts

```diff
@@ -352,6 +381,35 @@ export const translations: DefaultTranslations = {
       year_in_plural: 'på flere __count__ år'
     }
   },
+  pl: {
+    translation: {
+      now: 'neste exato momento',
```

Comment:
I'm no Polish speaking person, but this can't be Polish. According to Google Translate it's Portuguese

Reply:
Could perhaps be `czas` or `teraz`

Reply:
Good catch, that was indeed a copy&paste error. Should be fixed now.